weed/s3api/chunked_reader_v4.go

```diff
@@ -169,7 +233,9 @@ type s3ChunkedReader struct {
 	region            string
 	state             chunkState
 	lastChunk         bool
-	chunkSignature    string
+	chunkSignature    string // Empty string if unsigned streaming upload.
```

Comment:
`chunkSignature` is how we determine if we are `unsigned-streaming-upload` or the signed version. Not sure if we should use an enumeration or switch on the type. It will probably be needed in the future if we implement `SigV4A` types of signature.